components/ViewToolbar.vue

```diff
@@ -1,30 +1,34 @@
 <template>
-    <v-row dense :class="[$style.floatToolbar, 'flex-column']">
-        <v-col>
-            <v-tooltip location="left">
-                Reset camera
-                <template #activator="{ props }">
-                    <v-btn @click.stop=reset_camera icon="mdi-crop-free" density="comfortable" v-bind="props" />
-                </template>
-            </v-tooltip>
-        </v-col>
-    </v-row>
+  <v-row dense :class="[$style.floatToolbar, 'flex-column']">
+    <v-col>
+      <v-tooltip location="left">
+        Reset camera
+        <template #activator="{ props }">
+          <v-btn @click.stop=reset_camera density="comfortable" v-bind="props" icon>
+            <v-icon size="32">
```

Comment:
Pk mettre la v-icon plutot que la prop icon du v-btn ?

Reply:
It enables me to manage the icon's size wich was to small imo